test_config_files.py

```diff
@@ -11,7 +11,8 @@ def create_mock_settings():
     return settings
   
 def main():
-    feedback_pipeline.get_configs(create_mock_settings())
+    config_manager = content_resolver.config_manager.ConfigManager()
+    config_manager.get_configs(create_mock_settings())
```

Comment:
@Fan, this has somehow broke CR-input CI, e.g.:

https://github.com/minimization/content-resolver-input/actions/runs/12016353802/job/33496359657

Reply:
I just made a PR to fix this here https://github.com/minimization/content-resolver/pull/78